src/external/modules/returns/pre-handlers.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { returnStatuses } = require('shared/lib/constants');
+const returnPath = require('external/lib/return-path');
+
+const assertReturnStatusIsDue = (request, h) => {
+  if (!isDueReturn(request.model)) {
```

Comment:
It may be healthier if `isDueReturn` would just take `request` as the input.

Because.... (Go to second comment)